api/src/main/java/org/pmiops/workbench/config/CommonConfig.java

```diff
@@ -18,14 +19,13 @@ JsonFactory jsonFactory() {
     return new JacksonFactory();
   }
 
-  @Bean
-  HttpTransport httpTransport() {
-    return UrlFetchTransport.getDefaultInstance();
-  }
-
   @Bean
   Clock clock() { return Clock.systemUTC(); }
 
   @Bean
   Random random() { return new SecureRandom(); }
+  
+  @Bean
+  GoogleCredential.Builder googleCredentialBuilder() { return new GoogleCredential.Builder(); }
```

Comment:
Beans are singleton by default (surprised? I was too), so I don't think this will work as intended. Adding `@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)` should fix.